src/util.ts

```diff
@@ -1,3 +1,5 @@
+export const box = safeGetElementById("myContainer");
```

Comment:
Doesn't seem like this belongs exclusively to the game, so makes more sense to move it into `util.ts` for now.